### ISSUE TITLE
--kubeconfig arg does not seem to override environment setup

### ISSUE THREAD

**Issue by c-knowles** [wercker/stern#45]
### Steps to reproduce

1. Have access to at least two k8s clusters using stern 1.5.0
1. Run `export KUBECONFIG=PATH_TO_FIRST_KUBECONFIG`
1. Run `stern --kubeconfig PATH_TO_SECOND_KUBECONFIG --since 1m --namespace=kube-system kube-dns`

### Expected behaviour

CLI options override environment setup and stern shows logs based on config at `PATH_TO_SECOND_KUBECONFIG`. i.e. mirror the behaviour of `kubectl`.

I believe this stern used to do this but I've lost the version I had since I clean up brew. I think it was 1.3.0 or 1.2.0.

### Actual behaviour

stern show logs based on environment config at `PATH_TO_FIRST_KUBECONFIG`

**Comment by akupila** [comment]
Thanks for this & clear details on what's happening. Looks like there was a regression introduced in https://github.com/wercker/stern/commit/41697a5d72fdea156df74f26b865e242be5faa2c (released in 1.5.0). PR on the way to fix this

**Comment by andrewgdavis** [comment]
The override works correctly (ie --kubeconfig PATH_TO_SECOND_KUBECONFIG) but I'm not sure this is working as when using the default.
KUBECONFIG can be set to multiple config files... not just one, which results in an error like this
```
$GOPATH/bin/stern --all-namespaces kube-dns -c dnsmasq
failed to get client config: stat /Users/me/.kube/config:/Users/me/.kube/development-env/kube-config:/Users/me/vagrant/kubeconfig.yml: no such file or directory
```
It seems to me that it should operate similar to `kubectl config get-contexts` with `kubectl config current-context` as the default.
